pyrsa/data/dataset.py

```diff
@@ -14,6 +14,10 @@
 from pyrsa.util.descriptor_utils import bool_index
 from pyrsa.util.descriptor_utils import format_descriptor
 from pyrsa.util.descriptor_utils import parse_input_descriptor
+from pyrsa.util.file_io import write_dict_hdf5
```

Comment:
One way to make this a bit less verbose could be `import pyrsa.util.file_io as fio` and then `fio.write_dict_hdf5()`